package.json

```diff
@@ -12,6 +12,9 @@
 
   "dependencies": {
     "babel": "^5.8.23",
+    "bluebird": "^2.10.2",
```

Comment:
Is this dependency necessary? Or is the plan to utilize it for some `Promise.coroutine` magic? :smile: